slices/debianutils.yaml

```diff
@@ -0,0 +1,35 @@
+package: debianutils
+
+essential:
+  - debianutils_copyright
+
+slices:
+  bins:
+    essential:
+      - debianutils_config
+      - libc6_libs
```

Comment:
`dash_bins` can be added as a dependency here for the executables that are actually shell scripts.